index.bs

```diff
@@ -1367,7 +1368,7 @@ To <dfn>deserialize local value</dfn> given |realm| and |local protocol value|:
       1. If |value| does not match [=Date Time String Format=], return [=error=] with
          [=error code=] [=invalid argument=].
 
-      1. Return [=success=] with data [=Date.parse=](|value|).
+      1. Return [=success=] with data [=Date=]([=Date.parse=](|value|)).
```

Comment:
So this would call `Date()` as function and would return a string representation of the date. I'm sure that we actually want to have a Date object instead, which would require the usage of `new`. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#constructor

Reply:
Actually the `Date` constructor should hereby also work identical to the static `Date.parse()` method so that we may be able to actually get rid of `Date.parse()`.

Reply:
Ok, I just thought it's the way we write it in the specification, for example [here](https://w3c.github.io/webdriver-bidi/#deserialize-local-value) for the `Map` it's `Map(iterable)` instead of `new Map(iterable)`. I agree that's for `Date` it's confusing since it can be called as a function and constructor.
With `Date.parse`, yes, it's true, it's not really required, I'll remove it

Reply:
@jgraham looks like we need feedback in regards the usage of `new`. Could you tell us how should be used here?

Reply:
Well for the same operation HTML write `Otherwise, if serialized.[[Type]] is "Date", then set value to a new Date object in targetRealm whose [[DateValue]] internal slot value is serialized.[[DateValue]].`, which sidesteps the problem.

Since `Date` works differently when called as a function and as a constructor, I wonder if the spec-wise correct way to invoke it is using https://tc39.es/ecma262/#sec-construct and an assert that it returns a normal completion type.

Reply:
So then would it be something like 
```
2. Let "date_result" be "Construct(Date, [value])". If "date_result" has no a [[DateValue]] internal slot, return error with error code "invalid argument".
3. Return "success" with data "date_result".
```